packages/mui-material/src/CssBaseline/CssBaseline.js

```diff
@@ -43,6 +43,10 @@ export const styles = (theme, enableColorScheme = false) => {
     'strong, b': {
       fontWeight: theme.typography.fontWeightBold,
     },
+    a: {
+      color:
```

Comment:
```suggestion
      color: (theme.vars || theme).palette.primary.main,
```

I think `primary.main` should be the best one for links?

Reply:
I've followed @danilo-leal's pointer https://github.com/mui/material-ui/issues/38017#issuecomment-1642867004

Reply:
@siriwatknp Do you still want the change to reference `theme.vars` ?